--- /home/u/ws/canaan/rtos_k230/tools/gen_image_func.sh	2025-03-27 16:28:46.789856570 +0800
+++ ./tools/gen_image_func.sh	2025-04-27 09:19:08.116835624 +0800
@@ -30,7 +30,7 @@
 
 bin_gzip_ubootHead_firmHead()
 {
-	local mkimage="${SDK_UBOOT_BUILD_DIR}/tools/mkimage"
+	local mkimage="mkimage"
 	local file_full_path="$1"
 	local filename=$(basename ${file_full_path})
 	local mkimgArgs="$2"
@@ -46,26 +46,3 @@
 	rm -rf ${filename} ${filename}.gz ug_${filename}
 }
 
-# gz_file_add_ver()
-# {
-# 	[ $# -lt 1 ] && return
-# 	local f="$1"
-
-# 	local sdk_ver="v0.0.0";
-# 	local nncase_ver="0.0.0";
-# 	local sdk_ver_file="${K230_SDK_ROOT}/board/common/post_copy_rootfs/etc/version/release_version"
-# 	local nncase_ver_file="${K230_SDK_ROOT}/src/big/nncase/riscv64/nncase/include/nncase/version.h"
-# 	local storage="$(echo "$f" | sed -nE "s#[^-]*-([^\.]*).*#\1#p")"
-# 	local conf_name="${CONF%%_defconfig}"
-# 	local micropython_ver="v0.4";
-
-# 	micropython_ver="$(awk -F- '/^micropython/ { print $1}' ${sdk_ver_file} | cut -d: -f2 )" 
-# 	sdk_ver="$(awk -F- '/^sdk:/ { print $1}' ${sdk_ver_file} | cut -d: -f2 )"
-# 	cat ${nncase_ver_file} | grep NNCASE_VERSION -w | cut -d\" -f 2 > /dev/null && \
-# 		 nncase_ver=$(cat ${nncase_ver_file} | grep NNCASE_VERSION -w | cut -d\" -f 2)
-# 	rm -rf  ${conf_name}_${storage}_${sdk_ver}_nncase_v${nncase_ver}.img.gz;
-# 	rm -rf CanMV-K230_micropython_*;
-# 	ln -s  $f CanMV-K230_micropython_${micropython_ver}_sdk_${sdk_ver}_nncase_v${nncase_ver}.img.gz;
-# }
-
-
